home/modules/contribute/pages/generate-rest-api.adoc

```diff
@@ -0,0 +1,241 @@
+= Generate REST API from Swagger/OpenAPI
```

Comment:
Just a minor point, we (tech comms) don't generate the REST APIs, just their documentation

```suggestion
= Generate REST API Documentation
```